pkgs/development/perl-modules/IOCompressBrotli/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  brotli,
+  buildPerlPackage,
+  fetchurl,
+  perlPackages,
+  pkg-config,
+}:
+
+buildPerlPackage {
+  pname = "IO-Compress-Brotli";
+  version = "0.019";
+
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/T/TI/TIMLEGGE/IO-Compress-Brotli-0.019.tar.gz";
+    hash = "sha256-N/QN187kSs6iby92Onc+YdTsIjMF3e7KRhJEPL8oj78=";
+  };
+
+  patches = [
+    # Use system brotli in Makefile.PL
+    ./use-system-brotli.patch
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    brotli
+    perlPackages.FileSlurper
+    perlPackages.ExtUtilsPkgConfig
+  ];
+
+  propagatedBuildInputs = with perlPackages; [
+    FileSlurper
+    GetoptLong
+  ];
+
+  env = {
+    NIX_CFLAGS_COMPILE = "-I${brotli.dev}/include";
+    NIX_CFLAGS_LINK = "-L${lib.getLib brotli}/lib -lbrotlienc -lbrotlidec -lbrotlicommon";
```

Comment:
not sure where this is coming from

```
IO-Compress-Brotli> Warning (mostly harmless): No library found for -lbrotlienc-L/nix/store/pc94vsjs6d1d5z0a0ajbcckbjaif4r70-brotli-1.1.0-lib/lib
IO-Compress-Brotli> Warning (mostly harmless): No library found for -lbrotlidec-L/nix/store/pc94vsjs6d1d5z0a0ajbcckbjaif4r70-brotli-1.1.0-lib/lib
```

Reply:
Fixed by setting `LIBS => "-L${lib.getLib brotli}/lib -lbrotlienc -lbrotlidec -lbrotlicommon"` in Makefile.PL